ramls/hold.json

```diff
@@ -62,6 +62,10 @@
       "description": "Date the request was cancelled",
       "type": "string",
       "format": "date-time"
+    },
+    "patronComments": {
```

Comment:
We have to bump minor version for the API that uses this schema (in RAML and in ModuleDescriptor). We also require `circulation v9.5` API now (see ModuleDescriptor/requires section)

Reply:
I suspected that some point were missed, and as it turned out that was updating of dependencies. Thank you for notice this.